staging/src/k8s.io/component-helpers/storage/volume/pv_helpers.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package persistentvolume
```

Comment:
Can you double check if all the moved functions have at least more than one binary depending on it?

Reply:
I have checked this out and found nothing like that. The script I have used here:

```sh
make all GOLDFLAGS="" GOGCFLAGS=all="-N -l"
find _output/bin/ -type f | xargs -n1 sh -c 'gdb -batch -ex "info functions k8s.io/component-helpers/storage/volume" $0 2>/dev/null | awk -F"[.(]" -v FN=$0 "/void/{ print FN, \$4 }"' > /tmp/pkg_func_usage.txt
awk '{ print $2 }' /tmp/pkg_func_usage.txt | sort | uniq -c
```

The results here:
```
      4 CheckAccessModes
      6 CheckNodeAffinity
      4 CheckVolumeModeMismatches
      6 checkVolumeNodeAffinity
      4 claimToClaimKey
      4 FindMatchingVolume
      4 GetBindVolumeToClaim
      8 GetPersistentVolumeClaimClass
      6 GetPersistentVolumeClass
      4 IsDelayBindingMode
      3 IsDelayBindingProvisioning
      4 IsVolumeBoundToClaim
```

But I found `GetVolumeNodeAffinity` function which is used only by unit test `TestFindMatchVolumeWithNode`. I will move it closer to the test and make private.

Reply:
Oops, I found another place where `GetVolumeNodeAffinity` func is used. This is test too but it located in another package `k8s.io/kubernetes/pkg/scheduler/framework/plugins/volumebinding/binder_test.go`.

Reply:
@alculquicondor What do you think is it ok if I move this function in `k8s.io/kubernetes/staging/src/k8s.io/component-helpers/storage/volume/pv_helpers_test.go` and leave it public?

Reply:
No. Better copy it in each of the packages that need it. It's just a factory of the API. `Get` is not a very useful prefix. Maybe `build` is a better prefix.